host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent

```diff
@@ -0,0 +1,56 @@
+upstream glitchtip {
+   server localhost:8000;
+   keepalive 32;
+}
+
+server {
+    listen 80;
+    server_name glitchtip.pratchett.zeus.gent;
+
+    if ($request_method = GET) {
+        return 301 https://$server_name$request_uri;
+    }
+
+    return 308 https://$server_name$request_uri;
+}
+
+server {
+    listen 10443 ssl http2;
+    server_name glitchtip.pratchett.zeus.gent;
+
+    ssl_certificate     /etc/ssl/private/pratchett.zeus.gent.fullchain.pem;
+    ssl_certificate_key /etc/ssl/private/pratchett.zeus.gent.pem;
+    include snippets/ssl_options_preload.conf;
+
+   location / {
+       return 302 https://glitchtip.zeus.gent/;
+   }
+}
+
+server {
+    listen 10443 ssl http2;
+    server_name glitchtip.zeus.gent;
```

Comment:
should be glitchtip.pratchett.zeus.gent

Reply:
Should this be fixed here as well then? https://github.com/ZeusWPI/ansible-config/blob/master/host_files/pratchett/nginx/gamification.pratchett.zeus.gent#L32

Reply:
No, that's correct (although I now see that there should be an extra block with a port 80 http redirect to https for gamifcation.zeus.gent). The *.pratchett.zeus.gent are for "internal use" domains (we don't have to change the DNS config for them)